### DISCUSSION TITLE
Resolve feature catalog promotions

### DISCUSSION THREAD

**Discussion by vldmr-k** [lunarphp/lunar#2027] [Feature Requests]
Hello 
I've lot of expirience with magento/sylius, and I'm very tired from them. They are very overengineering. 
I found lunrar, and it's amazin simple but quite advance engine for e-commerce project.
Thanks for the author.

One feature which sylius has, it's https://docs.sylius.com/the-book/products/catalog-promotions

Is it possible to add this feature into lunar? 

Thanks

**Comment by glennjacobs** [comment]
Our Discounts system is the only thing available currently. But something along those lines I think would be a good feature to have in the future.

For now, you'd have to come up with something custom for your needs.

**Comment by supernrm** [comment]
+1, discounts alone aren't sufficient. Promotions are a must-have for e-commerce.

**Comment by olegmelnik** [comment]
+1
